pkg/fixtures/writer_plain_mysql.go

```diff
@@ -0,0 +1,81 @@
+package fixtures
+
+import (
+	"fmt"
+	"github.com/Masterminds/squirrel"
+	"github.com/applike/gosoline/pkg/cfg"
+	"github.com/applike/gosoline/pkg/db"
+	"github.com/applike/gosoline/pkg/mon"
+	"reflect"
+)
+
+type PlainMySqlFixtureValue []interface{}
+
+type PlainMySqlFixture struct {
+	TableName string
```

Comment:
usually this kind of metadata we set for the whole fixture writer instance and not per fixture (metadata that applies to all fixtures for this writer)

Reply:
Done.